Examples-Web/IdleTimeout/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import './App.scss';
+
+import { useEffect } from 'react';
+
+import CursorManager from 'Components/CursorManager';
+import IdleOverlay from 'Components/IdleOverlay';
+import UseIdleTimeout from 'Hooks/IdleTimeout';
+
+import { ConnectionManager } from './TouchFree/Connection/ConnectionManager';
+import { WebInputController } from './TouchFree/InputControllers/WebInputController';
+
+const App = () => {
+    useEffect(() => {
+        ConnectionManager.init();
+        const controller: WebInputController = new WebInputController();
+
+        new CursorManager();
+
+        return () => {
+            controller.disconnect();
+        };
+    }, []);
+
+    // Must be called after ConnectionManager.init()
+    const isIdle = UseIdleTimeout(10000);
```

Comment:
Might be worth dropping this down a little, just as it's a demo and it's nice to not have to wait too long to see it!